haalcentraal/haalcentraal-brp/src/main/resources/config/liquibase/changelog/20220413-rename-connector-to-haalcentraal-brp.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright 2015-2022 Ritense BV, the Netherlands.
+  ~
+  ~ Licensed under EUPL, Version 1.2 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" basis,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="Ritense" id="1">
+        <update tableName="connector_type">
+            <column name="name" value="HaalCentraalBrp" type="VARCHAR(512)"/>
+            <where>name = 'HaalCentraal'</where>
+        </update>
+    </changeSet>
+
+    <changeSet author="Ritense" id="2">
+        <update tableName="connector_type">
+            <column name="connector_properties" type="${jsonType}" valueComputed="REPLACE(connector_properties, 'com.ritense.haalcentraal.connector.HaalCentraalBRPProperties', 'com.ritense.haalcentraal.brp.connector.HaalCentraalBrpProperties')" />
```

Comment:
Does this also work for postgres?

Reply:
[It should](https://www.postgresqltutorial.com/postgresql-string-functions/postgresql-replace/)

But I've only tested it for mysql